x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -187,7 +187,7 @@ export class FleetPlugin
   private kibanaVersion: FleetAppContext['kibanaVersion'];
   private kibanaBranch: FleetAppContext['kibanaBranch'];
   private httpSetup?: HttpServiceSetup;
-  private securitySetup?: SecurityPluginSetup;
+  private securitySetup!: SecurityPluginSetup;
```

Comment:
Adding this operator to fix typescript error `TS2564: Property 'name' has no initializer and is not definitely assigned in the constructor.`

Reply:
should we keep the `?` ? as this property could potentially not be initialized if the setup method is not called. It should not happen while running Kibana but when using this service in tests.

Reply:
I think the reasoning is that these properties should be aligned with whatever is in `kibana.json`, so I changed it because in production we'll always call this setup. 
I see your point about tests, in that case both `securitySetup` and `securityStart` should be kept optional. I have no strong preference about it :)

Reply:
`securitySetup!` is good it will throw if it's not set in tests